### PR TITLE
Update prev Project setup page and links to content

### PR DESCRIPTION
**Pull request by heatheramz**
#### Description of changes: 
1 - Updates prev version Project setup to be on one page since cannot use menu to navigate. Example: https://prev-versions-project-setup-callouts.d9kfkg9g23gld.amplifyapp.com/swift/prev/start/project-setup/
2 - Updates some lib-v1 fragments to align headers on combined Project setup page
3 - Updates redirects from old site for prev Project setup to point to the updated page.

### Instructions

**If this PR should not be merged upon approval for any reason, please submit as a DRAFT**

Which product(s) are affected by this PR (if applicable)?
- [ ] amplify-cli
- [ ] amplify-ui
- [ ] amplify-studio
- [ ] amplify-hosting
- [ ] amplify-libraries

Which platform(s) are affected by this PR (if applicable)?
- [ ] JS
- [ ] iOS
- [ ] Android
- [ ] Flutter
- [ ] React Native

**Please add the product(s)/platform(s) affected to the PR title**

#### Checks

- [ ] Does this PR conform to [the styleguide](https://github.com/aws-amplify/docs/blob/main/STYLEGUIDE.md)?

- [ ] Does this PR include filetypes other than markdown or images? Please add or update unit tests accordingly.

- [ ] Are any files being deleted with this PR? If so, have the needed redirects been created?

- [ ] Are all links in MDX files using the MDX link syntax rather than HTML link syntax? <br /> 
      _ref: MDX: `[link](https://link.com)` 
            HTML: `<a href="https://link.com">link</a>`_

### When this PR is ready to merge, please check the box below
- [ ] Ready to merge

_By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license._

### CODE REVIEW

File: src/fragments/lib-v1/project-setup/ios/combine/combine.mdx [hunk 1]
```diff
@@ -71,11 +71,11 @@ struct MyAppCode {
 }
 ```
 
-## Installation
+### Installation
 
 There is no additional work needed to enable Combine support. Projects that declare a deployment target of iOS 13.0 or higher will automatically see the appropriate method signatures and properties, depending on the Category and API you are calling.
 
-## API Comparison
+### API Comparison
```
Comment (hibler13) [inline src/fragments/lib-v1/project-setup/ios/combine/combine.mdx:78]: Suggest we lowercase "comparison" here.

File: src/pages/[platform]/prev/start/project-setup/index.mdx [hunk 2]
```diff
@@ -1,37 +1,230 @@
...
     }
   };
 }
 
-<Overview childPageNodes={props.childPageNodes} />
+<Callout info>
+
+We recommend that you use the [latest version for project setup](/[platform]/start/project-setup/) for your chosen framework or language. 
+
+This page is a reference for those who need the previous version and includes all project setup content for that version. Clicking the navigation menu will take you back to the current version by default.
```
Comment (hibler13) [inline src/pages/[platform]/prev/start/project-setup/index.mdx:38]: For accessibility purposes, we should avoid "clicking." Change to "Selecting the navigation menu..."

File: src/pages/[platform]/start/project-setup/prerequisites/index.mdx [hunk 3]
```diff
@@ -19,6 +19,12 @@ export function getStaticProps(context) {
   };
 }
 
+<Callout info>
+
+Project setup guides are available for the [previous version](/[platform]/prev/start/project-setup/prerequisites/). However, it is highly recommended that you use the latest version to set up new projects following this guide.
```
Comment (hibler13) [inline src/pages/[platform]/start/project-setup/prerequisites/index.mdx:24]: Change second sentence to "However, we highly recommend using the latest version and following this guide when setting up new projects."

**Review by hibler13 — COMMENTED** [review]
@heatheramz Had a few minor editorial changes, but otherwise LGTM!

File: src/fragments/lib-v1/project-setup/ios/combine/combine.mdx (hunk 1 above)
Comment (heatheramz) [inline src/fragments/lib-v1/project-setup/ios/combine/combine.mdx:78]:
```suggestion
### API comparison
```

File: src/pages/[platform]/prev/start/project-setup/index.mdx (hunk 2 above)
Comment (heatheramz) [inline src/pages/[platform]/prev/start/project-setup/index.mdx:38]:
```suggestion
This page is a reference for those who need the previous version and includes all project setup content for that version. Selecting the navigation menu will take you back to the current version by default.
```

File: src/pages/[platform]/start/project-setup/prerequisites/index.mdx (hunk 3 above)
Comment (heatheramz) [inline src/pages/[platform]/start/project-setup/prerequisites/index.mdx:24]:
```suggestion
Project setup guides are available for the [previous version](/[platform]/prev/start/project-setup/prerequisites/). However, we highly recommend using the latest version and following this guide when setting up new projects.
```

**Review by heatheramz — COMMENTED** [review]
Capturing changes

**Comment by heatheramz** [comment]
After further review we decided to get rid of the following update - "Updated sections without versioning with new callout for users. Updated Start, Getting Started, Project Setup, Build UI, Deploy and Host, and Tools landing pages. Example: https://prev-versions-project-setup-callouts.d9kfkg9g23gld.amplifyapp.com/swift/start/" Will look for another solution instead.

**Review by aspittel — APPROVED** [review]

**Review by katiegoines — APPROVED** [review]

**Pull request merged**